Model/IO/Export/Nastran/BulkData/ShortBulkData.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by Kiełbasa Karol on 28.03.2017.
+//
+
+#ifndef CPP_SHORTBULKDATA_H
+#define CPP_SHORTBULKDATA_H
+#pragma once
```

Comment:
Explicit guard header or pragma once - choose one. AFAIK pragma once is not yet in standard.